### DISCUSSION TITLE
访问权限不允许套接字尝试

### DISCUSSION THREAD

**Discussion by GTEMYee** [Meloong-Git/PCL#1788] [讨论]
如题，附图
群友的pcl
![-3260a74be7de7967](https://user-images.githubusercontent.com/117887221/235937492-042061cc-53d7-4bce-9e88-fd786596f65b.png)

**Comment by Chiloven945** [comment]
~你这ip直接出来了……~
请附上log文件，随后交一个issue

**Reply by SqrtTan** [reply]
IP？DDoS！(bushi

**Reply by hejiehao** [reply]
NAT不需要担心

**Comment by Pigeon0v0** [comment]
上次碰到这个问题是重装系统解决的...

**Comment by Yue-cn** [comment]
v6 or v4
